src/services/openai.ts

```diff
@@ -1,33 +1,32 @@
 import { LocalStorageKeys } from '../../const'
-
-export const getOpenAIResponse = async (prompt: string) => {
+import type { ChatCompletionCreateParams } from 'openai/resources/chat/completions'
+type PartialChatParams = Omit<
+  ChatCompletionCreateParams,
+  'model' | 'temperature'
+> & {
+  model?: string
+  temperature?: number
+}
+export const getOpenAIResponse = async (config: PartialChatParams) => {
   try {
-    const { openai_api_key, openai_ai_model } = await browser.storage.local.get(
-      [LocalStorageKeys.OPENAI_API_KEY, LocalStorageKeys.OPENAI_AI_MODEL]
-    )
+    const { openai_api_key } = await browser.storage.local.get([
+      LocalStorageKeys.OPENAI_API_KEY,
+    ])
```

Comment:
are we intending to remove the OPENAI_AI_MODEL setting as now getOpenAIResponse will use gpt-4o if the caller doesn't specify